### PR TITLE
Remove commands (oc adm router and oc adm create-node-config)

### PR DESCRIPTION
**Pull request by soltysh**
It's about time we drop these since they are useless in 4.0. 

/assign @deads2k @smarterclayton

### CODE REVIEW

File: pkg/oc/cli/admin/admin.go [hunk 1]
```diff
@@ -127,9 +126,7 @@ func NewCommandAdmin(name, fullName string, f kcmdutil.Factory, streams genericc
 	// Deprecated commands that are bundled with the binary but not displayed to end users directly
 	deprecatedCommands := []*cobra.Command{
 		// these will be removed soon
-		admin.NewCommandNodeConfig(admin.NodeConfigCommandName, fullName+" "+admin.NodeConfigCommandName, streams),
 		node.NewCommandManageNode(f, node.ManageNodeCommandName, fullName+" "+node.ManageNodeCommandName, streams),
-		router.NewCmdRouter(f, fullName, "router", streams),
 		ipfailover.NewCmdIPFailoverConfig(f, fullName, "ipfailover", streams),
```
Comment (soltysh) [inline pkg/oc/cli/admin/admin.go]: @smarterclayton do we want to get rid of these and below ca-related commands too?

File: pkg/oc/cli/admin/admin.go [hunk 2]
```diff
@@ -127,9 +126,7 @@ func NewCommandAdmin(name, fullName string, f kcmdutil.Factory, streams genericc
 	// Deprecated commands that are bundled with the binary but not displayed to end users directly
 	deprecatedCommands := []*cobra.Command{
 		// these will be removed soon
-		admin.NewCommandNodeConfig(admin.NodeConfigCommandName, fullName+" "+admin.NodeConfigCommandName, streams),
 		node.NewCommandManageNode(f, node.ManageNodeCommandName, fullName+" "+node.ManageNodeCommandName, streams),
```
Comment (smarterclayton) [inline pkg/oc/cli/admin/admin.go]: Kill manage node as well (can be separate PR)

File: pkg/cmd/server/admin/create_client.go [hunk 3]
```diff
@@ -177,3 +177,20 @@ func (o CreateClientOptions) CreateClientFolder() error {
 
 	return nil
 }
+
+// readFiles returns a byte array containing the contents of all the given filenames,
```
Comment (smarterclayton) [inline pkg/cmd/server/admin/create_client.go]: who uses this?

**Comment by smarterclayton** [comment]
Just a few comments

File: pkg/cmd/server/admin/create_client.go (hunk 3 above)
Comment (soltysh) [reply pkg/cmd/server/admin/create_client.go]: Other commands from the same package, but it might die with all them being removed.

File: pkg/oc/cli/admin/admin.go (hunk 2 above)
Comment (soltysh) [reply pkg/oc/cli/admin/admin.go]: Yup, that's the plan.

Comment (soltysh) [reply pkg/oc/cli/admin/admin.go]: This and ipfailover too.

**Comment by soltysh** [comment]
Fixed cmd and verify, re-tagging.
